app/views/content/non-uk-teachers/visas-for-non-uk-teachers.md

```diff
@@ -99,25 +99,25 @@ The salary you must be paid to get a visa varies according to  whether you are e
 
 If you are employed as a qualified teacher and work in:
 
-* inner London, you must earn £34,502
-* outer London, you must earn £32,407
-* London fringe, you must earn £29,344
-* the rest of England, you must earn £28,000
+* inner London, you must earn £36,745
+* outer London, you must earn £34,514
+* London fringe, you must earn £31,350
+* the rest of England, you must earn £30,000
 
 If you are employed as an unqualified teacher and work in:
 
-* inner London, you must earn £24,254
-* outer London, you must earn £22,924
-* London fringe, you must earn £20,960
-* the rest of England, you must earn £20,960
+* inner London, you must earn £25,831
+* outer London, you must earn £24,415
+* London fringe, you must earn £23,200
+* the rest of England, you must earn £23,200
```

Comment:
@laurahermionetennant92 this is the same amount as for London fringe. Is that correct?